cobigen-templates/templates-devon4j/src/main/templates/crud_java_server_app_complex/templates.xml

```diff
@@ -1,59 +1,50 @@
-                       destinationPath="src/main/java/${variables.rootPackage}/${variables.component}/logic/impl/${variables.component#cap_first}Impl.java"
-                       mergeStrategy="javamerge"/>
-    <templateExtension ref="crud_complex_${variables.component#cap_first}Ctos.java" 
-                       destinationPath="../api/src/main/java/${variables.rootPackage}/${variables.component}/logic/api/${variables.component#cap_first}.java"
-                       mergeStrategy="javamerge"/>
-                       
-    <templateExtension ref="crud_complex_AbstractLogic.java" mergeStrategy="javamerge"/>
-    
-    
+    <templateExtension ref="crud_complex_UcFind${variables.entityName}Ctos.java" destinationPath="../api/src/main/java/${variables.rootPackage}/${variables.component}/logic/api/usecase/UcFind${variables.entityName}.java" mergeStrategy="javamerge" />
+    <templateExtension ref="crud_complex_UcFind${variables.entityName}ImplCtos.java" destinationPath="src/main/java/${variables.rootPackage}/${variables.component}/logic/impl/usecase/UcFind${variables.entityName}Impl.java" mergeStrategy="javamerge" />
+    <templateExtension ref="crud_complex_${variables.component#cap_first}ImplCtos.java" destinationPath="src/main/java/${variables.rootPackage}/${variables.component}/logic/impl/${variables.component#cap_first}Impl.java" mergeStrategy="javamerge" />
+    <templateExtension ref="crud_complex_${variables.component#cap_first}Ctos.java" destinationPath="../api/src/main/java/${variables.rootPackage}/${variables.component}/logic/api/${variables.component#cap_first}.java" mergeStrategy="javamerge" />
+
+    <templateExtension ref="crud_complex_AbstractLogic.java" mergeStrategy="javamerge" />
+
+
   </templates>
 
   <templateScans>
-    <templateScan templateNamePrefix="crud_complex_" templatePath="templates" destinationPath="src/main"/>
+    <templateScan templateNamePrefix="crud_complex_" templatePath="templates" destinationPath="src/main" />
   </templateScans>
 
   <increments>
-    <increment name="logic_impl_complex" description="CRUD UC logic">
-      <templateRef ref="crud_complex_AbstractBeanMapperSupport.java"/>
-      <templateRef ref="crud_complex_${variables.component#cap_first}.java"/>
-      <templateRef ref="crud_complex_Abstract${variables.entityName}Uc.java"/>
-      <templateRef ref="crud_complex_AbstractUc.java"/>
-      <templateRef ref="crud_complex_AbstractLogic.java"/>
-      <templateRef ref="crud_complex_${variables.component#cap_first}Impl.java"/>
-      <templateRef ref="crud_complex_UcFind${variables.entityName}.java"/>
-      <templateRef ref="crud_complex_UcManage${variables.entityName}.java"/>
-      <templateRef ref="crud_complex_UcFind${variables.entityName}Impl.java"/>
-      <templateRef ref="crud_complex_UcManage${variables.entityName}Impl.java"/>
+    <increment name="logic_impl_complex" description="CRUD UC logic" explanation="Creates (backends) server CRUD service for java application.">
```

Comment:
I would mention that this increment mainly creates the logic layer of the Java application, which is then used in the actual REST service